clients/csharp-dotnet-core/voice-assistant-test/README.md

```diff
@@ -174,14 +174,41 @@ Here is the full list:
 >>##### ExpectedResponses 
 >>`JSON string | optional`. An array of Bot-Framework JSON activities. These are the expected bot responses. You only need to specify the activity fields you care about. If the number of bot responses is less than what you specify here, the test will fail. If the number is the same, but some of the fields you specify do not match the fields in the bot reply, the test will fail. Note that the tool orders the bot responses based on their activity [Timestamp](https://github.com/Microsoft/botframework-sdk/blob/master/specs/botframework-activity/botframework-activity.md#timestamp), before comparing the actual bot response to the expected response. If the number of bot responses is greater than the expected number, and there is a match to all expected activities, the test will pass. If ExpectedResponses is null or missing, the turn will run in bootstrapping mode, which is useful to do when writing the tests for the first time. See the [bootstrapping mode](#bootstrapping-mode) section for more details.
 >>
->>##### ExpectedTTSAudioResponseDuration 
->>`array of integers | optional | null | [1500, -1, 2000]`. Expected duration (in msec) of bot response TTS audio stream. This allows you to validate that the right audio stream duration was downloaded by the tool. Otherwise the test will fail. The length of this array (if exists) must match the length of the [ExpectedResponses](#expectedresponses) array. Not every bot-response will have a TTS audio stream associated with it. In that case, specify a value of -1 in the array cell. The expected duration does not have to exactly match the actual duration for the test to succeed. This is controlled by the field [TTSAudioDurationMargin](#ttsaudiodurationmargin). Only if the different between expected duration and actual duration is outside this margin, the test will fail. See the [Getting Started Guide](docs/GETTING-STARTED-GUIDE.md) for an example of using ExpectedTTSAudioResponseDuration.
+>>##### ExpectedTTSAudioResponseDurations 
+>>`array of strings | optional | null | ["1500", "-1", "2000"]`. Expected duration (in msec) of bot response TTS audio stream. This allows you to validate that the right audio stream duration was downloaded by the tool. Otherwise the test will fail. The length of this array (if exists) must match the length of the [ExpectedResponses](#expectedresponses) array. Not every bot-response will have a TTS audio stream associated with it. In that case, specify a value of -1 in the array cell. The expected duration does not have to exactly match the actual duration for the test to succeed. This is controlled by the field [TTSAudioDurationMargin](#ttsaudiodurationmargin). Only if the different between expected duration and actual duration is outside this margin, the test will fail. See the [Getting Started Guide](docs/GETTING-STARTED-GUIDE.md) for an example of using ExpectedTTSAudioResponseDurations.

```

Comment:
Instead of ["1500", "-1", "2000"] you can add an example with ||, something line "["1500", "-1", "2000 || 1700"]

Reply:
Done.